src/test/java/com/wikia/webdriver/common/clicktracking/events/EventsArticleEditMode.java

```diff
@@ -1,10 +1,10 @@
 package com.wikia.webdriver.common.clicktracking.events;
 
+import com.wikia.webdriver.common.clicktracking.EventParameter;
+
 import javax.json.Json;
 import javax.json.JsonObject;
 
-import com.wikia.webdriver.common.clicktracking.EventParameter;
-
 public class EventsArticleEditMode {
 
   private EventsArticleEditMode() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)